src/main/java/com/chariotsolutions/meetupnotifier/AppBean.java

```diff
@@ -0,0 +1,5 @@
+package com.chariotsolutions.meetupnotifier;
+
+public interface AppBean {
```

Comment:
I would change `AppBean` to `MeetupNotifier`and `AppBeanImpl` to `MeetupNotifierImpl`